utils/parsing.go

```diff
@@ -312,3 +312,26 @@ func GetUniqueCohortDefinitionIdsList(cohortDefinitionIds []int, filterCohortPai
 	uniqueIdsList := MakeUnique(idsList)
 	return uniqueIdsList
 }
+
+func Intersect(list1 []int, list2 []int) []int {
+	result := []int{}
```

Comment:
maybe use set for result so no need to make unique and then convert to list if need?

Reply:
thanks. Pushed up a faster implementation, as discussed.